packages/iiif-preview-server/src/stores/netlify-blob.ts

```diff
@@ -1,26 +1,27 @@
 import { getStore } from "@netlify/blobs";
-import { StorageInterface } from "../types";
+import type { StorageInterface } from "../types";
 
 export function createNetlifyStore(
   config: {
     defaultTtl?: number;
     netlifyStoreName?: string;
-  } = {}
+  } = {},
 ): StorageInterface {
   const manifests = getStore(config.netlifyStoreName || "preview-manifests");
 
   return {
     get: async (key: string) => {
-      const itemString = await manifests.get(key);
-      const item = itemString ? JSON.parse(itemString) : null;
+      const resource = await manifests.getWithMetadata(key);
+      const metadata = (resource?.metadata || {}) as any;
```

Comment:
The netlify blob has a metadata property, with the `ttl` so we don't need to store it in the JSON itself. This worked for the setting, but not the updating.